docs/contributing/templates/partner-app-otel.md

```diff
@@ -0,0 +1,95 @@
+---
+id: partner-app-otel-doc
+title: Partner App (OTEL) Template
+sidebar_label: Partner App (OTEL) Template
+description: Using the Sumo Logic Partner App (OTEL) Template.
+---
+
+:::tip
+Copy this file and edit it for your App integration.
+:::
+
+<img src={useBaseUrl('img/integrations/folder-name/app-name.png')} alt="icon" width="75"/>
+
+Introduction
+
+<img src='Architecture-S3-image-link' style={{border: '1px solid black'}} alt="<App Name> OTel Collection architecture" />
+
+## Fields creation in Sumo Logic for \<App Name\>
+
+Following are the tags which will be created as part of \<App Name\> App install if not already present. 
+
+## Prerequisites
+
+This section provides instructions for configuring metrics and log collection for the Sumo Logic app for \<App Name\>.
+
+### Metric collection
+
+### Log collection
+
+## Collection configuration and app installation
+
+{@import ../../../reuse/apps/opentelemetry/config-app-install.md}
```

Comment:
These import plugins are causing the PR to fail because the path is one level too deep.

For this template file, it'd be `{@import ../../reuse/apps/opentelemetry/config-app-install.md}`, but since the real files will live under `/integrations/<app category>`, the path you have here is correct. 

I would propose maybe commenting these out, as I've done [here](https://raw.githubusercontent.com/SumoLogic/sumologic-documentation/main/docs/contributing/templates/partner-app.md).